static/src/js/routes/Admin/Admin.js

```diff
@@ -55,11 +56,20 @@ export class Admin extends Component {
                   >
                     View Projects
                   </PortalLinkContainer>
+                  |
+                  {' '}
+                  <PortalLinkContainer
+                    to="/admin/retrievalsMetrics"
+                  >
+                    View Retrieval Metrics
+                  </PortalLinkContainer>
                 </Route>
                 <Route exact path={`${path}/retrievals`} component={AdminRetrievalsContainer} />
                 <Route exact path={`${path}/retrievals/:id`} component={AdminRetrievalContainer} />
                 <Route exact path={`${path}/projects`} component={AdminProjectsContainer} />
                 <Route exact path={`${path}/projects/:id`} component={AdminProjectContainer} />
+                <Route exact path={`${path}/projects/:id`} component={AdminProjectContainer} />
```

Comment:
Is line a duplicate of the line above it?

Reply:
Yes mistake removed